src/test/routes/sign-in.test.ts

```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import { app } from '../../main/app';
+
+describe('Sign In option', () => {
```

Comment:
you are missing all post and redirect cases

Reply:
done